doc/sphinxman/source/build_planning.rst

```diff
@@ -1655,25 +1657,27 @@ the examples below, *testname* are regex of :source:`test names <tests>`,
 and *testlabel* are regex of labels (*e.g.*, ``cc``, ``mints``,
 ``libefp`` defined :source:`[here, for example] <tests/ci-property/CMakeLists.txt#L3>` .
 
-* Run tests in parallel with ``-j`` flag. For maximum parallelism: :samp:`ctest -j\`getconf _NPROCESSORS_ONLN\`\ `
+* Run tests in parallel with ``-j`` flag. For maximum parallelism (not Windows): :samp:`ctest -j\`getconf _NPROCESSORS_ONLN\`\ `
 * Run full test suite: ``ctest``
 * Run about a third of the tests in 5 minutes, the so-called *quicktests*: ``ctest -L quick``
-* Run the same subset of tests that TravisCI checks (not the full test suite): ``ctest -L quick``
+* Run the same subset of tests that continuous integration checks (not the full test suite): ``ctest -L quick``
 * Run the minimal number of tests to ensure Psi4 and any add-ons in working order: ``ctest -L smoke``
 * Run tests matching by name: ``ctest -R testname``
 * Run tests excluding those by name: ``ctest -E testname``
 * Run tests matching by label: ``ctest -L testlabel``
 * Run tests excluding those by label: ``ctest -LE testlabel``
 
-Pytest has similar commands
-* Run tests in parallel with ``-n`` flag (if extenstion pytest-xdist installed). For maximum parallelism: :samp:`pytest -n\`getconf _NPROCESSORS_ONLN\`\ `
+Pytest has similar commands:
```

Comment:
I think mentioning the `-v` and `-vv` options for pytest might be helpful, as the default printing of failed tests in pytest is pretty awful. With `-vv` you get a full diff (at least for the numpy comparison functions).